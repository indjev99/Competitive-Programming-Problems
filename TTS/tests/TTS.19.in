78 2
-10000 -5645.59
-9442.53 -10000
790.66 10000
4762.5 -10000
10000 -9494.03
-10000 -1236.74
-10000 6498.33
10000 7771.08
-9595.64 -10000
6558.46 -10000
-10000 -4558.04
10000 9471.83
10000 2208.32
-2482.27 -10000
8692.8 -10000
10000 3164.65
-10000 -8788.7
7845.28 -10000
10000 4523.55
4584.62 -10000
-7879.5 10000
-6404.87 -10000
10000 -8191.85
10000 5012.93
-10000 7215.99
-10000 -8985.3
7472.66 -10000
-10000 1295.57
-2770.64 10000
10000 9177.34
-10000 -7977.12
-10000 6201.47
-9734.97 -10000
-5758.87 10000
9054.84 10000
10000 -4521.61
10000 4465.85
-10000 715.36
-3860.48 10000
10000 -3013.65
1287.6 10000
10000 -582.88
10000 -6180.96
10000 6332.19
10000 -6826.82
3035.88 -10000
10000 9882.48
-4448.59 10000
-10000 852.66
10000 -5999.59
-472.11 10000
10000 815.6
10000 2455.84
-8190.6 10000
-10000 5574.66
-10000 7344.15
5514.1 10000
-1888.71 -10000
3599.72 -10000
5766.43 10000
-6352.26 10000
8388.11 10000
9452.25 -10000
-3787.64 -10000
4197.55 -10000
-10000 -7901.02
-10000 -6737.94
10000 3415.74
-10000 5422.63
8355.29 -10000
7081.94 10000
-485.46 10000
-8620.93 -10000
10000 -4777.76
-10000 8457.81
10000 -5892.3
5368.63 10000
10000 -6934.32
6780.2 7350.43
-3850.2 9229.08
