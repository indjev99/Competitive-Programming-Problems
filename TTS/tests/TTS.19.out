3420.490000
